### PR TITLE
Cl hunchentoot half

### PR DESCRIPTION
**Pull request by catap**
#### Description

This is the first half of https://github.com/macports/macports-ports/pull/18819

@herbygillot 

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by catap** [comment]
@mascguy, @herbygillot this one is ready quite a long time. May I ask you to merge it? :)

File: lisp/cl-ppcre/Portfile [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           common_lisp 1.0
+
+github.setup        edicl cl-ppcre 2.1.1 v
+revision            1
```
Comment (mascguy) [inline lisp/cl-ppcre/Portfile:8]: I know this is nit-picky, but set your revision to zero.  ;-)

No worries though, I know it's tough when you're juggling this many ports in one PR!

Comment (catap) [reply lisp/cl-ppcre/Portfile:8]: @mascguy it isn't. I've moved original port and removed variants, and heavy reworked it ;)

Comment (mascguy) [reply lisp/cl-ppcre/Portfile:8]: Ah gotcha, no worries!

**Comment by mascguy** [comment]
Looks good overall, I don't see any obvious issues.

My only thought/suggestion, relates to the portgroup: Ideally we should be encapsulating all of our pgs using namespaces, as is done in `debug` and others. (Though that's certainly not a requirement, so no worries if you prefer it as-is.)

Up to you though, I'm fine with whatever you prefer! And you can certainly update it in the future, if/when you find time. Your thoughts?

**Review by mascguy — APPROVED** [review]
LGTM, this collection of ports is a great addition!

**Comment by catap** [comment]
@mascguy thanks for suggestion.

Anyway, as soon as I've fixed another lisps this PG should be adjusted, and when I'll do that I'll wrap functions into namespaces ;)

**Pull request merged**